ts/store/reducers/notifications/localScheduled.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Local scheduled notifications reducer
+ */
+
+import { getType } from "typesafe-actions";
+
+import {
+  clearLocalNotificationsScheduled,
+  updateLocalNotificationsScheduled
+} from "../../actions/notifications";
+import { Action } from "../../actions/types";
+import { GlobalState } from "../types";
+
+export type LocalScheduledState = Readonly<{
+  isToSchedule: boolean;
```

Comment:
this is a little confusing since it's negated (is not scheduled), I would turn it into positive to make it easier to understand:

```suggestion
  isScheduled: boolean;
```

Reply:
I fixed it in the commit [3c874ae](https://github.com/teamdigitale/io-app/pull/992/commits/3c874aee998ce81853bd3bdc26d881f855192ef1)